tests/test_transpile_self.py

```diff
@@ -199,7 +199,7 @@ def test_cpp_recursive(self):
 
     def test_julia_recursive(self):
         settings = self.SETTINGS["julia"]
-        suppress_exceptions = False if SHOW_ERRORS else (NotImplementedError,)
+        suppress_exceptions = (False,) if SHOW_ERRORS else (NotImplementedError,)
```

Comment:
I wish we could be a bit stricter with typing `suppress_exceptions`. I'll merge this as-is for now. Let's clean up later.